evap/evaluation/models.py

```diff
@@ -162,6 +163,12 @@ def general_questionnaires(self):
     def contributor_questionnaires(self):
         return super().get_queryset().filter(type=Questionnaire.Type.CONTRIBUTOR)
 
+    def dropout_questionnaires(self):
```

Comment:
If you're touching these and it's no trouble, it would probably be nice if you could also add the return type annotation (`-> QuerySet[Questionnaire]` or whatever it should be)

(same for the views below that return `HttpResponse`. It's a bit annoying, but for backwards compatibility, no annotation always means "Any", even if the method always returns some specific type. I wonder if [mypy issue 4409](https://www.github.com/python/mypy/issues/4409) will one day be implemented.)